tests/python/frontend/onnx/test_graph.py

```diff
@@ -5,8 +5,8 @@
 from model_zoo import super_resolution
 
 def compare_graph(onnx_file, nnvm_sym, ishape):
-    onnx_graph = onnx.load(onnx_file)
-    onnx_sym, params = nnvm.frontend.from_onnx(onnx_graph)
+    onnx_model = onnx.load(onnx_file)
+    onnx_sym, params = nnvm.frontend.from_onnx(onnx_model.graph)
```

Comment:
I think a property would be check onnx_model.version and then change the code accordingly. So the current test pass and we can merge things in gradually

Reply:
Neat Idea. I will try.

Reply:
Well, what do you think about the version of `onnx`, not a model but the imported module? If you fix it to v0.2, you should update the `onnx` installed on the testing environment.

Reply:
The older `onnx` (that is installed on the testing environment) will try to load a proto file as `GraphProto`, and v0.2 is going to load one as 'ModelProto'.

Reply:
The current test is built upon 0.1. I am just suggesting we do things like try and get ir version field of proto before we proceed so code works for both versions and the transition can be smooth. Otherwise we can update the env and the importer at the same time, but need to grantee test pass when merging

Reply:
As far as reading [this issue on ONNX project](https://github.com/onnx/onnx/issues/119), versioning of an ONNX model is quite fragile at the moment. So an alternative is to check the version of ONNX module and change the code accordingly.

Reply:
Sound good to me

Reply:
can you start the change so that test can pass and we can merge the updates in?

Reply:
yes, I'm working on it.

Reply:
ah... the primary version of ONNX is devastating...